routeguide/client/src/main/scala/handlers/RouteGuideClientHandler.scala

```diff
@@ -42,8 +42,8 @@ class RouteGuideClientHandler[F[_]: ConcurrentEffect](
         .use(_.getFeature(Point(lat, lon)))
         .map { feature: Feature =>
           if (feature.valid)
-            logger.info(s"Found feature called '${feature.name}' at ${feature.location.pretty}")
-          else logger.info(s"Found no feature at ${feature.location.pretty}")
+            logger.info(s"Found feature called '${feature.name}' at ${feature.location.get.pretty}")
```

Comment:
I'd avoid the usage of `[Option].get`

Reply:
Ah yeah that's probably a bad habit I picked up from my old job... what do you recommend I use instead?  Should I try folding or something?

Reply:
Right 👍

Reply:
updated! :)